runtime/src/lib.rs

```diff
@@ -532,20 +534,43 @@ impl Default for ProxyType {
 
 impl InstanceFilter<Call> for ProxyType {
 	fn filter(&self, c: &Call) -> bool {
-		if let Call::Sudo(pallet_sudo::Call::sudo { call, .. }) = c {
-			if let Call::Domains(pallet_domains::Call::force_register_domain { .. }) = &**call {
-				return true;
-			}
-		}
-
 		match self {
 			ProxyType::Any => true,
-			ProxyType::DomainRegistrar => matches!(
+			ProxyType::DomainRegistrar => {
+				if let Call::Sudo(pallet_sudo::Call::sudo { call, .. }) = c {
+					return matches!(
+						&**call,
+						Call::Domains(pallet_domains::Call::force_register_domain { .. })
+					)
+				}
+				false
+			},
+			// We don't include any space or profile management pallets actions here.
```

Comment:
I am not sure abut this comment. It is already clear that here are no spaces and profiles pallet for this type of proxy. Why to explain what is already clear?